Sources/CoffeeAddictsLibrary/Helpers/CoffeeShopsLocationHelper.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mihai Garda Popescu on 23.07.2021.
+//
+
+import Foundation
+import CoreLocation
+
+class CoffeeShopsLocationHelper {
```

Comment:
Ha a `helper` class (see my comment relate to the utils folder), I will argue that this classes don't need to exist, for example this class does at least 2 things, some stuff relate do distance and validation.

One can pull a specialized class that does validation easily and even build a nice dsl for it.

What do you think? Are there good use cases for 'helper' classes?

Reply:
> Ha a `helper` class (see my comment relate to the utils folder), I will argue that this classes don't need to exist, for example this class does at least 2 things, some stuff relate do distance and validation.
> 
> One can pull a specialized class that does validation easily and even build a nice dsl for it.
> 
> What do you think? Are there good use cases for 'helper' classes?

Replied to this in the general comment